modules/led.sh

```diff
@@ -231,6 +238,21 @@ function newyear {
   led red "$current_red"
 }
 
+function newmoonyear {
+  current_green=$(led "green")
+  current_red=$(led "red")
+
+  for i in 1 2 3 4 5 6 7 8
+  do
+  set_brightness 1 0 && set_brightness 1 1 sleep 0.1
```

Comment:
So `set_brightness 1 0` turns the red led off while `set_brightness 1 1` turns the red led on.

Which means that the line `set_brightness 1 0 && set_brightness 1 1 sleep 0.1` doesn't actually do anything (the off and on commands are executing at the same time, effectively cancelling each other).

If you wanted to make if flash, you would have to do something like:
```
set_brightness 1 0 && sleep 0.1  # turns red off and sleeps for 0.1 seconds
set_brightness 1 1 && sleep 0.1  # turns red on and sleeps for 0.1
```

You have to remember that after you turn a led off, unless you explicitly turn them back on again, they will stay off.

Reply:
Also need to update help files and other files for this new addition.